Gems/Atom/Tools/MaterialCanvas/Code/Source/Document/MaterialCanvasDocument.h

```diff
@@ -22,15 +22,16 @@
 
 namespace MaterialCanvas
 {
-    //! MaterialCanvasDocument
+    //! MaterialCanvasDocument implements support for creating, loading, saving, and manipulating graph model and canvas graphs that
+    //! represent and will be transformed into shader and material data.
     class MaterialCanvasDocument
         : public AtomToolsFramework::AtomToolsDocument
         , public MaterialCanvasDocumentRequestBus::Handler
         , public GraphModelIntegration::GraphControllerNotificationBus::Handler
         , public GraphCanvas::SceneNotificationBus::Handler
     {
     public:
-        AZ_RTTI(MaterialCanvasDocument, "{90299628-AD02-4FEB-9527-7278FA2817AD}", AtomToolsFramework::AtomToolsDocument);
+        AZ_RTTI(MaterialCanvasDocument, "{C60CA7B2-D9FA-4E20-AD7D-D8A661902A7D}", AtomToolsFramework::AtomToolsDocument);
```

Comment:
Looks like it was using the same ID as MaterialDocument.
Are there any assets already checked in the repo using the old ID?

Reply:
I thought I already updated this but just in case I did it again. The document classes aren't actually serialized. They manage the serialization of whatever they contain. So the UUID would not have affected anything in that sense.